boards/arm/arduino_portenta_h7/arduino_portenta_h7-common.dtsi

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2022 Benjamin Björnsson <benjamin.bjornsson@gmail.com>.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	gpio_keys {
+		compatible = "gpio-keys";
+		wake_up: button {
+			label = "wakeup";
+			gpios = <&gpioc 13 GPIO_ACTIVE_HIGH>;
```

Comment:
Which button is referred here?
I have checked the schematics and for me, the button on the Portenta is directly connected to `NRST` and the one on the Breakout Board is connected to the PMIC `PWRON`.

Reply:
You're right, this should be removed.